### PR TITLE
Tier-1 SSE-S3 feature automation.

### PR DESCRIPTION
**Pull request by viduship**
**Tier-1 SSE-S3  feature automation**.

**Automates Polarian TCs :** CEPH-83574615, CEPH-83574617, CEPH-83575151, CEPH-83574619, CEPH-83575150 and CEPH-83575153

Logs:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-EOPQZK

Signed-off-by: viduship <vimishra@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by anrao19** [comment]
@viduship Could please add complete suite log to identify the stage?

File: suites/pacific/rgw/tier-2_sse_s3_encryption.yaml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+  - test:
+      config:
+        script-name: test_ss3_s3_with_vault.py
+        config-file-name: test_sse_s3_per_bucket_encryption_normal_object_upload.yaml
+        timeout: 300
+      desc: test_sse_s3_per_bucket_encryption_normal_object_upload
+      module: sanity_rgw.py
+      name: sse-s3 per bucket encryption test
+      polarion-id: CEPH-83574615, CEPH-83574617, CEPH-83575151
```
Comment (udaysk23) [inline suites/pacific/rgw/tier-2_sse_s3_encryption.yaml:97]: @viduship AFAIK, we are not allowed to mention multiple polarion ids(I may wrong here). Could you please check on this

Comment (viduship) [reply suites/pacific/rgw/tier-2_sse_s3_encryption.yaml]: @udaysk23, It has been addressed.

**Review by TejasC88 — APPROVED** [review]

File: pipeline/metadata/6.0.yaml [hunk 2]
```diff
@@ -229,6 +229,22 @@ suites:
       - rgw
       - stage-1
 
+  - name: "test-rgw-sse-s3-encryption-GA"
+    suite: "suites/pacific/rgw/tier-2_sse_s3_encryption.yaml"
+    global-conf: "conf/pacific/rgw/tier-0_rgw.yaml"
```
Comment (udaysk23) [inline pipeline/metadata/6.0.yaml:234]: @viduship it looks like path is incorrect here and we might not able to list this suite in custom jenkins run while executing it for 6.x build

**Pull request merged**